### PR TITLE
Made conditional for drone launch more readable

### PR DESCRIPTION
**Pull request by NickyBobby**
I looked around and didn't see much else that needed to be refactored. Just this conditional to make it more readable.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -37,6 +37,10 @@ def set_top_categories
     @top_categories ||= Category.take(4)
   end
 
+  def launch_drone?
+    params[:drone].to_i == 3
+  end
+
```
Comment (amaxwellblair) [inline app/controllers/application_controller.rb]: I think we should make this a helper method or put it in the orders helper @NickyBobby

**Comment by NickyBobby** [comment]
You want me to make an orders helper in the same fashion as the stores helper? @amaxwellblair

**Comment by NickyBobby** [comment]
I added the OrdersHelper module. Let me know what you think. @amaxwellblair @ksk5280

**Comment by amaxwellblair** [comment]
Looks good to me!

**Pull request merged**